ankihub/main/note_type_management.py

```diff
@@ -32,12 +36,21 @@ def update_note_type_fields(note_type: NotetypeDict, fields: List[str]) -> None:
     print("update_note_type_fields", note_type["name"], fields)
 
 
+def remove_ankihub_end_comment_from_note_type(note_type: Dict[str, Any]) -> None:
+    note_type["css"] = ANKIHUB_CSS_END_COMMENT_PATTERN.sub("", note_type["css"])
+    for template in note_type["tmpls"]:
+        template["qfmt"] = ANKIHUB_HTML_END_COMMENT_PATTERN.sub("", template["qfmt"])
+        template["afmt"] = ANKIHUB_HTML_END_COMMENT_PATTERN.sub("", template["afmt"])
```

Comment:
Good catch, I forgot about that.

:question: Shouldn't we create a copy of the note type before modifying it? I worry that we could change the note type in Anki's `ModelManager` and that this could cause issues.
We could change the function to `note_type_with_ankihub_end_comment_removed` and make it return a modified copy of the note type.

Reply:
Great catch. I never worried about modifying note types returned by ModelManager but this can cause issues because they are cached.

Reply:
🤔 Maybe it'd be good to make a change to Anki to make `ModelManager.get` return a copy?

Reply:
I agree. Will suggest a PR.